gauntlet/packages/gauntlet-solana-contracts/artifacts/schemas/serum_multisig.json

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
Hard-add it here or let it be added along other programs and .so's during Gauntlet setup?

Reply:
Let's have it here, in the future will be pulled from GH. But for development is useful